### PR TITLE
Update protobuf version for master

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
Bumping to the latest released ballerina protobuf version
## Checklist
- [ ] ~Linked to an issue~
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -43,7 +43,7 @@ stdlibMimeVersion=2.4.0
 stdlibTaskVersion=2.2.3-20220920-184600-c109457
 stdlibUrlVersion=2.2.2
 stdlibRandomVersion=1.3.0
-stdlibProtobufVersion=1.3.1-20221006-101100-9a65fe3
+stdlibProtobufVersion=1.3.2-20221006-142100-0685135
```
Comment (niveathika) [inline gradle.properties:46]: Latest released version seems to be 1.3.1. Any reason to update this to the next developement timestamped version

Comment (dilanSachi) [reply gradle.properties:46]: I was going to release grpc patch for 2201.2.x with protobuf:1.3.1 version here in https://github.com/ballerina-platform/module-ballerina-grpc/pull/1029

We can use the same version here as well right?

Comment (niveathika) [reply gradle.properties:46]: Yes the version is 1.3.1. Here you have updated to `1.3.2`

Comment (dilanSachi) [reply gradle.properties]: Updated

**Pull request merged**